model/src/main/kotlin/io/spine/validation/PatternPolicy.kt

```diff
@@ -79,16 +77,14 @@ internal class PatternPolicy : Policy<FieldOptionDiscovered>() {
     }
 }
 
-private fun checkFieldType(field: Field, file: File) {
-    val type = field.type
-    if (!(type.isSingularString || type.isRepeatedString)) {
-        Compilation.error(file, field.span) {
-            "The field type `${field.type}` of `${field.qualifiedName}` is not supported " +
-                    "by the `($PATTERN)` option. Supported field types: strings and repeated " +
-                    "of strings."
-        }
+private fun checkFieldType(field: Field, file: File) =
+    Compilation.check(field.type.isSupported(), file, field.span) {
+        "The field type `${field.type}` of `${field.qualifiedName}` is not supported " +
```

Comment:
Let's have the convention for new lines in concatenations: "They start from the space character". It's easier to spot and understand as a continuation. Oftentimes, people miss a space character in concatenated messages. With the starting space it happens less often.

Reply:
Please also add this as a rule in our Code Style Wiki.

Reply:
I've added this convention for [Kotlin](https://github.com/SpineEventEngine/documentation/wiki/Kotlin-code-style#formatting-string-literals) and [Java](https://github.com/SpineEventEngine/documentation/wiki/Java-code-style#formatting-string-literals) styles.